src/client/services/feature/feature.js

```diff
@@ -61,7 +62,7 @@ var FeatureService = SparkBase.extend(
     return this.request({
       method: 'POST',
       api: 'feature',
-      resource: 'features/users/' + this.spark.device.userId + '/developer',
```

Comment:
There are tests for this. Please determine why tests didn't catch this and update accordingly.